alf/trainers/policy_trainer.py

```diff
@@ -164,13 +164,13 @@ def _train(self):
             t = time.time() - t0
             logging.log_every_n_seconds(
                 logging.INFO,
-                '%s: %s time=%.3f throughput=%0.2f' %
-                (common.get_gin_file(), iter_num, t, int(train_steps) / t),
+                '%s -> %s: %s time=%.3f throughput=%0.2f' %
+                (common.get_gin_file(), [os.path.basename(self._root_dir)],
```

Comment:
Why do you need a list around the directory here?

Reply:
Just to make the printed information looks good, more symmetric around ```->```.